virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc

```diff
@@ -0,0 +1,32 @@
+[id="virt-configuring-pci-passthrough"]
+= Configuring PCI passthrough
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-configuring-pci-passthrough
+
+toc::[]
+
+//This assembly contains the content for
+//configuring PCI passthrough by using the CLI. There are
+//plans to enable PCI passthrough configuration
+//by using the web console (next release).
+//When this feature is available in the web console, please
+//add the new content to this assembly.
+
+include::modules/virt-about-pci-passthrough.adoc[leveloffset=+1]
+include::modules/virt-adding-kernel-arguments-enable-iommu.adoc[leveloffset=+2]
+include::modules/virt-binding-devices-vfio-driver.adoc[leveloffset=+2]
+include::modules/virt-adding-pci-device-hyperconverged-operator.adoc[leveloffset=+2]
+
+[id="virt-configuring-vms-for-pci-passthrough"]
+== Configuring virtual machines for PCI passthrough
+
+After the PCI devices have been added to the cluster, you can assign them to virtual machines. The PCI devices are now available as if they are physically connected to the virtual machines.
+
+include::modules/virt-assigning-pci-device-virtual-machine.adoc[leveloffset=+2]
+
+[id="additional-resources_configuring_pci_passthrough"]
```

Comment:
This might not be a big deal, but the only underscore should be between `resources` and `configuring`; the others should be hyphens

Reply:
Updated the content.